packages/main/src/components/SplitterElement/index.tsx

```diff
@@ -51,20 +51,49 @@ export interface SplitterElementPropTypes extends CommonProps {
  */
 const SplitterElement = forwardRef((props: SplitterElementPropTypes, ref: RefObject<HTMLDivElement>) => {
   const { children, style, tooltip, className, minSize, size, resizable, ...rest } = props;
+  const [componentRef, splitterElementRef] = useSyncRef(ref);
+  const { vertical, reset } = useContext(SplitterLayoutContext);
+  const [flex, setFlex] = useState(size !== 'auto' ? `0 0 ${size}` : '1 0 min-content');
+  const [flexBasisApplied, setFlexBasisApplied] = useState(false);
```

Comment:
I'm not 100% why we need the second state here - can't we derive it from size and reset?